tests/keras_tests/feature_networks_tests/feature_networks/scale_equalization_test.py

```diff
@@ -45,11 +46,11 @@ def get_quantization_config(self):
                                       weights_per_channel_threshold=True,
                                       activation_channel_equalization=True)
 
-    def create_inputs_shape(self):
+    def get_input_shapes(self):
```

Comment:
This test fails due to the small batch size.
maybe we should change the compare function? (remove cosine similarity) and add another test?